### ISSUE TITLE
There are bugs when using Chinese

### ISSUE THREAD

**Issue by xiusin** [serkonda7/termtable#35]
```
+---------+-----------------+
| 键     | 值             |
+---------+-----------------+
| V版本 | V 0.2.2 0dafdb4 |
+---------+-----------------+
```

**Comment by serkonda7** [comment]
Yeah, this is already a known issue (#27) because the implementation is blocked by a missing feature in V itself.
The width of CJK letters is not calculated correctly.

Recently someone made progress there, but I lack free time right now to check if it works for termtable.
Feel free to make a PR on your own or you might have to wait a few weeks :(

**Comment by serkonda7** [comment]
Closing as duplicate of #27

**Comment by serkonda7** [comment]
@xiusin I have implemented support for chinese. It was actually a lot easier and faster than I thought.

I would love, if you could give it a try and give feedback :heart:
